subsys/event_manager/event_manager.c

```diff
@@ -60,13 +61,17 @@ static void log_event(const struct event_header *eh)
 		return;
 	}
 
-	if (et->log_event) {
-		if (CONFIG_EVENT_MANAGER_EVENT_LOG_BUF_LEN > 0) {
-			log_event_using_buffer(eh, et);
-		} else {
-			(void)et->log_event(eh, NULL, 0);
-		}
-	} else if (IS_ENABLED(CONFIG_EVENT_MANAGER_LOG_EVENT_TYPE)) {
+#if !IS_ENABLED(CONFIG_EVENT_MANAGER_USE_DEPRECATED_LOG_FUN)
+	if (et->log_event_func) {
+		et->log_event_func(eh);
+	}
+#else
```

Comment:
Shouldn't we support both cases here? It seems like only deprecated way of logging is supported here.